src/Studio/Widgets/ChannelMultiSelectionWidget.h

```diff
@@ -13,10 +13,10 @@
 **
 ** neuromore Public License Usage
 ** Alternatively, this file may be used under the terms of the neuromore
-** Public License version 1 as published by neuromore co with exceptions as 
-** appearing in the file neuromore-class-exception.md included in the 
-** packaging of this file. Please review the following information to 
-** ensure the neuromore Public License requirements will be met: 
```

Comment:
Really shouldn't be any changes to license - only place this is allowed in node exception